### PR TITLE
make booking_policy.rb file

### PR DESCRIPTION
**Pull request by chihoko12**
I made the file "booking_policy.rb" and added some authorization into the "bookings_controller.rb"

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-  before_action :set_booking, only: [:show, :edit, :update]
-  before_action :set_boat, only: [:new, :create]
+  before_action :find_booking, only: [:show, :edit, :update]
+  before_action :find_boat, only: [:new, :create]
   before_action :authenticate_user!, only: [:new, :create]
+  skip_before_action :authenticate_user!, only: [:index, :show]
 
   def index
-    @bookings = Booking.all
+    @bookings = policy(Booking).order(name: :asc)
```
Comment (sorina-duca) [inline app/controllers/bookings_controller.rb:8]: I think it is policy_scope(Booking). Bookings don't have a name, so I don't think we can order them alphabetically by name. I believe that will throw an error. Please let's test the code locally before we do the pull request.

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -23,27 +26,29 @@ def create
...
   def show
+    authorize @booking
   end
 
   def edit
+    authorize @booking
   end
 
   def update
     @booking.update(booking_params)
```
Comment (sorina-duca) [inline app/controllers/bookings_controller.rb:37]: I think we also need to authorize the update as well

Comment (chihoko12) [reply app/controllers/bookings_controller.rb:38]: thank you. add the authorization to the update method

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (chihoko12) [reply app/controllers/bookings_controller.rb]: thank you. i corrected it by sorting by check-in date.

**Pull request merged**